project/src/main/ui/menu/practice-menu.gd

```diff
@@ -1,147 +1,199 @@
 extends Control
-## Scene which lets the player repeatedly play a set of levels.
```

Comment:
Add top-level comment.

Reply:
Added comment, `## Scene which lets the player repeatedly play a set of levels.`